interoperator/pkg/internal/renderer/gotemplate/functions_test.go

```diff
@@ -36,21 +36,4 @@ func TestGoTemplateFunctions(t *testing.T) {
 	invalidObj2, err2 := unmarshalJSON(invalidObjStr2)
 	g.Expect(invalidObj2).To(gomega.BeNil())
 	g.Expect(err2).To(gomega.HaveOccurred())
-
```

Comment:
Can we keep the tests? Even though it will test the library, but I guess that is fine. WDYT?

Reply:
Kept and Updated the tests the `quote`, `squote` and `strval` fn from the libarary